### ISSUE TITLE
Issue when importing room users&messages from JSONL source

### ISSUE THREAD

**Issue by tsaruks** [mattermost/docs#2442]
Hi,
now we are under way on migratoion from HipChat to local Mattermost instance.
HipChat data converted by Hipmost utility.

We are successfully imported direct messages using this method but have issue with importing room's data. 
CLI command finished without errors but new room I've found only from CLI command line. I able to see new room in Mattermost only after manipulation with CLI command line:
1. manually add himself to new team
2. manually add himself to channel
But after this I've found that new room is empty, without any message. However source JSOL file contains many messages (see attached file).

Please help me resolve this situation.
Sergiy

[Dev chat.zip](https://github.com/mattermost/docs/files/2695449/Dev.chat.zip)

**Comment by amyblais** [comment]
Closing this as a duplicate of the Zendesk ticket you have opened,
